### ISSUE TITLE
Add needed commands from book to docker image

### ISSUE THREAD

**Issue by czabriskie** [adipolak/ml-with-apache-spark#1]
Hey thanks so much for putting this material together. I've really enjoyed the overview so far. Could we add to the Docker images the necessary things for the book? I'm sure many people, including me, won't want to have to add the datasets, files, and install pytorch everytime we want to work through the material.

It would be pretty easy. For the first half of the book, we could do something like this, adding the directory we clone from this github

```
FROM adipolak/ml-with-apache-spark

ADD ./dir /home/jovyan
```

For the second half of the book, I wasn't able to install `pytorch` and `torchvision` ontop of your image, but maybe we could use pytorch image as the base?

In both cases, dir would be the files from this github repo.

**Comment by adipolak** [comment]
yes, ofcourse. let me take a closer look into it. thank you for taking the time!
